website/blog/2018-06-27-supporting-jest-open-source.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Supporting Jest Open Source
+author: Rick Hanlon II
+authorURL: https://twitter.com/rickhanlonii
+authorFBID: 160800566
+---
+
+Jest is maintained by a combination of Facebook employees and a community of open source contributors.
```

Comment:
Can you reorder to "combination of community and Facebook employees"? Only 11% of the contributions over the last month came from Facebook folks and you guys go first.